### ISSUE TITLE
A11y_Visual Studio Code Client_Search_Global Search_Keyboard: We are not able to access the tooltips present for "Match case", "Match whole word" etc present in "global search" section via keyboard.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#38218]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

--- GitHubTags:
-A11yTCS; -A11ySev2; -DesktopApp; -MAC; -Visual Studio Code Client; -WCAG2.1.1; -Keyboard; -BM-VisualStudioCodeClient-MAC-Jan2024

--- **Environment details:**
Visual Studio Code insider version 1.86.0
macOS: Sonoma 14.2.1

--- Repro Steps:

1. Open Visual Studio Code Insiders editor
2. Using keyboard navigation View Menu Bar
3. TAB till search option and press ENTER.
4. TAB till "Match case", "Match whole word" etc controls.
5. Observed that We are not able to access the tooltips present for "Match case", "Match whole word" etc present in "global search" section via keyboard.

--- Actual Result:
We are not able to access the tooltips present for "Match case", "Match whole word" etc present in "global search" section via keyboard.

**Similar issue is also observed throughout the application.**

--- Expected Result:
We should be able to access the tooltips present for "Match case", "Match whole word" etc present in "global search" section via keyboard.

--- User Impact:
Keyboard only users will not be able to get tooltip information of the control.

--- Attachment:

https://github-com/microsoft/vscode/assets/143686145/51b903a7-62ae-4a14-9988-82db48ac8424
